client/src/containers/CampaignAdmin/CampaignAdmin.js

```diff
@@ -22,19 +22,22 @@ class CampaignAdmin extends SilverStripeComponent {
   constructor(props) {
     super(props);
 
-    this.publishApi = backend.createEndpointFetcher({
-      url: this.props.sectionConfig.publishEndpoint.url,
-      method: this.props.sectionConfig.publishEndpoint.method,
-      defaultData: { SecurityID: this.props.securityId },
-      payloadSchema: {
-        id: { urlReplacement: ':id', remove: true },
-      },
-    });
+    this.publishApi = (this.props.sectionConfig && this.props.sectionConfig.publishEndpoint)
```

Comment:
publishEndpoint should be statically there right?

when did sectionConfig become optional? and unit tests should be mocking sectionConfig